test/gaugeTest.js

```diff
@@ -43,19 +42,21 @@ describe('gauge', () => {
 			});
 
 			it('should start a timer and set a gauge to elapsed in seconds', () => {
-				const clock = lolex.install();
+				jest.useFakeTimers('modern');
```

Comment:
passing `modern` uses `@sinonjs/fake-timers` under the hood. It'll be the default from Jest 27